src/main/java/nl/tudelft/jpacman/annotations/ParameterizedAssignment.java

```diff
@@ -0,0 +1,8 @@
+package nl.tudelft.jpacman.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.CLASS)
+public @interface ParameterizedAssignment {
```

Comment:
With one parameterised assignment out there, isn't this redundant with `@Parameterized`? Didn't we aim for a more generic annotation, such as `@Assignment(Parameterized)` (using assignment enum value)?